content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md

```diff
@@ -12,6 +12,10 @@ Download the Cumulocity IoT Edge version 10.15 from the [Software AG Empower por
 
 Starting with Cumulocity IoT Edge Release 10.15, accessing the Edge appliance remotely from the Cumulocity IoT tenant using the Telnet protocol has been removed. Software AG recommends you to use the SSH protocol instead.
 
+### Delay in installation and update
```

Comment:
- [ ] Is the heading right by calling it as a "Delay"? We may have to find better wording.

Reply:
Updated the heading to "Installation and update processes take more time"